pkg/controller/reconciler/postgres_group_reconciler.go

```diff
@@ -0,0 +1,32 @@
+package reconciler
+
+import (
+	clientCore "github.com/oslokommune/okctl/pkg/client/core"
+	"github.com/oslokommune/okctl/pkg/controller/resourcetree"
+)
+
+// PostgresGroupReconciler handles reconciliation for dummy nodes (e.g. the root node) and acts as a template for other
+// reconcilers
+type PostgresGroupReconciler struct{}
```

Comment:
This reconciler does nothing. Why not use the [NoopReconciler](https://github.com/oslokommune/okctl/blob/master/pkg/controller/reconciler/noop.go) and a [group type](https://github.com/oslokommune/okctl/blob/master/pkg/controller/resourcetree/tree.go#L19) to group the resources? Thats what they're for. Or are they lacking something?

Reply:
I noticed them, but did not know how to use them properly :D

Reply:
Gahhh. My bad. Should be exactly the same as you used this